Makefile

```diff
@@ -1,7 +1,7 @@
 
 .PHONY: help
 help:
-	@echo "Usage: make [target]\n\nTargets:"
+	@echo "Usage: make [target] [ARGS='additional args']\n\nTargets:"
```

Comment:
Pulling in these changes from https://github.com/canonical/jubilant/pull/54 (reviewable there)